### PR TITLE
Setup Python 2/3 compatibility and support multiple Django versions

### PR DESCRIPTION
**Pull request by Duologic**
* Build on Travis-CI for multiple Python and Django versions

Related to https://vikingco.atlassian.net/browse/MVGR-787

### CODE REVIEW

File: smsgateway/backends/jasmin.py [hunk 1]
```diff
@@ -42,11 +48,11 @@ def send(self, sms_request, account_dict):
             url = self.get_send_url(request, account_dict)
 
             # Make request to provider
-            result = u''
+            result = ''
             if url is not None:
                 try:
                     sock = urlopen(url)
-                    result = sock.read()
+                    result = sock.read().decode('utf_8')
```
Comment (mhemeryck) [inline smsgateway/backends/jasmin.py]: Side note: `utf-8` should be default, so not really required here. Given that we're playing with encodings, it makes sense though.

File: smsgateway/smpplib/client.py [hunk 2]
```diff
@@ -232,7 +234,7 @@ def read_pdu(self):
 
         if p.is_error():
             raise Exception('({}) {}: {}'.format(p.status, p.command, descs[p.status]))
-        elif p.command in state_setters.keys():
+        elif p.command in list(state_setters.keys()):
```
Comment (mhemeryck) [inline smsgateway/smpplib/client.py]: I guess this was auto-fixed, because this is not really required

**Comment by JonasSteur** [comment]
@Duologic I'd say merge and release 📦

**Comment by Duologic** [comment]
Rebased, squashed and merged.

**Pull request closed without merging**